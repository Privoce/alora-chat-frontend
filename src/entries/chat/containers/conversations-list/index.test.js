import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import {
	ConversationsList
} from 'entries/chat/containers';

describe('<ConversationsList />', () => {
	const modelI = {
		items: [
			{
				color: 'red'
			}
		]
	};

	const getWrapper = (model) => {
		return shallow((
			<ConversationsList {...model} />
		));
	};

	it('renders correctly', () => {
		const wrapper = getWrapper(modelI);
		expect(toJson(wrapper)).toMatchSnapshot();
	});
});
