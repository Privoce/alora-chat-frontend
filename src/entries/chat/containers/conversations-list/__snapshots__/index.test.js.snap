// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ConversationsList /> renders correctly 1`] = `
<div
  className="conversations-list-container"
>
  <div>
    <ConversationCardComponent
      actions={
        Object {
          "options": Array [
            Object {
              "event": [Function],
              "text": undefined,
            },
          ],
        }
      }
      desc={
        Object {
          "fontSize": 13,
          "margin": "0px 0px 0px 14px",
          "maxWidth": 76,
          "text": undefined,
        }
      }
      key="0"
      onClick={[Function]}
      profile={
        Object {
          "backgroundColor": undefined,
          "color": "white",
          "height": 40,
          "label": undefined,
          "labelFontSize": 12,
          "width": 40,
        }
      }
      title={
        Object {
          "fontSize": 13,
          "margin": "0px 0px 0px 14px",
          "maxWidth": 76,
          "text": undefined,
        }
      }
    />
  </div>
</div>
`;
